examples/arch.rs

```diff
@@ -6,7 +6,13 @@
 use platform_info::*;
 
 fn main() {
-    let info = PlatformInfo::new().unwrap();
+    let info = match PlatformInfo::new() {
```

Comment:
You don't prefer this syntax?

```
let info = PlatformInfo::new().unwrap_or_else(|err| {
    eprintln!("Unable to determine platform info: {}", err);
    std::process::exit(1);
});
```

Reply:
Sure, I think that's more succinct.
I just created this code in a quick, more verbose, idiom.
I'll change it.

Reply:
I made the change in this location. I did leave the later `match ...` code to show an example of an alternate code construction.